go.mod

```diff
@@ -26,22 +25,20 @@ require (
 	github.com/decred/dcrd/wire v1.2.0
 	github.com/decred/dcrwallet/wallet v1.1.0
 	github.com/decred/slog v1.0.0
-	github.com/dgraph-io/badger v1.5.5-0.20190124015931-be03c12d2da2
+	github.com/dgraph-io/badger v1.5.5-0.20190214192501-3196cc1d7a5f
```

Comment:
badger update has removed global logger, which should not be used. We stopped using it previously, but this will prevent accidentally going back